docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+  
+# Dapr 0.11.0
+
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
+
+With Dapr approaching v1.0, our releases increasingly focus on operational concerns like stability, reliability, and upgradability in production environments.
+As always, community and user requests and feedback are top of mind along with bug fixes.
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+* Support for declarative mechanism to create topic subscription ([Declarative Subscriptions](https://github.com/dapr/docs/blob/master/howto/consume-topic/README.md#declarative-subscriptions))
+* [New access control lists for Service Invocation](https://github.com/dapr/docs/tree/master/concepts/service-invocation#service-access-security)
```

Comment:
I would rephrase to: `New access control policies and SPIFFE based identities for Service Invocation`